spot-client/src/spot-remote/remote-control/middleware.js

```diff
@@ -0,0 +1,20 @@
+import { MiddlewareRegistry } from 'common/redux';
+import { remoteControlService } from 'common/remote-control';
+
+import { SUBMIT_FEEDBACK } from './action-types';
+
+MiddlewareRegistry.register(() => next => action => {
+    const result = next(action);
+
+    switch (action.type) {
+    case SUBMIT_FEEDBACK: {
+        remoteControlService.submitFeedback({
+            score: action.score,
+            message: action.message
+        });
+        break;
+    }
+    }
```

Comment:
So the next step after all this work settles would be  bring (maybe) all rcs logic calls into here?

Reply:
yeah, I'm thinking all calls done by the UI should be converted to be actions.